app/src/androidTest/java/com/droidhats/campuscompass/views/CalendarFragmentTest.kt

```diff
@@ -149,7 +149,7 @@ class CalendarFragmentTest {
         //checks if there exists a button with text "okay" within the calendar selection widget an performs a click on it
         val appCompatButton2 = onView(
             allOf(
-                withId(android.R.id.button1), withText("Okay"),
+                withId(android.R.id.button1), withText("OK"),
```

Comment:
Change "OK" to "OKAY". It fixes the failure at this line for this test

Reply:
@JeremiahTee On my end, this is actually OK and not OKAY. Maybe on newer PR this has been reworded so far with respect to master, it's actually called OK and passes the test.